statusboard/src/pages/Projects/Projects.tsx

```diff
@@ -104,6 +105,16 @@ function Projects(): JSX.Element {
               selected={timeRange}
               options={Object.keys(ACTIVE_THRESHOLDS)}
             />
+            <Checkbox 
+              label="Display non-brigade projects?"
+              id="non_brigade_projects"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                setFilters({ nonCfA: String(e.target.checked) })
+              }
+              // I don't know why it's asking me these 2 parameters below
+              className="gio"
+              inline="inline"
```

Comment:
On line 5-10 in `Checkbox.js` you have made them required. You can give them default values, same as `Select.defaultProps` or by using ES6 features (suggested above). They will still give prop types warnings for now, but it should compile.

```suggestion
```